<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintSetup</class>
 <widget class="QWidget" name="PrintSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Print Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>24</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pages</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="Templates">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Create a default layout</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="AddPage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Append a page</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="InsertPage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Insert a page before the selected page</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="DeletePage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Delete the selected page</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="KComboBox" name="PageSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select a paper size</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="KComboBox" name="Orientation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select the paper orientation</string>
     </property>
     <item>
      <property name="text">
       <string>Portrait</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Landscape</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="KComboBox" name="Zoom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Zoom factor</string>
     </property>
     <item>
      <property name="text">
       <string notr="true">25%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">50%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">100%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">150%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">200%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">400%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">800%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">1600%</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="9">
    <widget class="QToolButton" name="SelectElement">
     <property name="toolTip">
      <string>Select an element</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QToolButton" name="TextElement">
     <property name="toolTip">
      <string>Insert a text element</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QToolButton" name="PatternElement">
     <property name="toolTip">
      <string>Insert a pattern element</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QToolButton" name="ImageElement">
     <property name="toolTip">
      <string>Insert a picture element</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QToolButton" name="KeyElement">
     <property name="toolTip">
      <string>Insert a key element</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QListWidget" name="Pages">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>214</red>
           <green>210</green>
           <blue>208</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>214</red>
           <green>210</green>
           <blue>208</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>244</red>
           <green>244</green>
           <blue>244</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string>Page list</string>
     </property>
     <property name="iconSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="gridSize">
      <size>
       <width>170</width>
       <height>170</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="5" colspan="9">
    <widget class="QScrollArea" name="PagePreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>100</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Page preview</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>0</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="14">
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
